lib/Dancer2/Core/Route.pm

```diff
@@ -9,6 +9,14 @@ use Scalar::Util 'blessed';
 
 our ( $REQUEST, $RESPONSE, $RESPONDER, $WRITER, $ERROR_HANDLER );
 
+my $count = 0;
+
+has name => (
+    is      => 'ro',
+    isa     => Str,
+    default => sub { $count++ },
```

Comment:
I suggest `++$count` so that the autogenerated names are always truthy (until ints wrap around :) ).  (I'm not sure when I would test `if($route->name)`, but somebody might, and truthy default names would prevent the first route declared from behaving differently from the others.)

Reply:
Ok. :+1:

Reply:
👍

Reply:
Resolved --- no longer relevant since routes are no longer given automatic names.